examples/nlp/joint_intent_slot_infer.py

```diff
@@ -19,12 +19,11 @@
                     default="bert-base-uncased",
                     type=str)
 parser.add_argument("--dataset_name", default='snips-all', type=str)
-parser.add_argument("--data_dir",
-                    default='data/nlu/snips',
-                    type=str)
+parser.add_argument("--data_dir", default='data/nlu/snips', type=str)
 parser.add_argument("--work_dir",
-                    default='outputs/SNIPS-ALL/20191010-164934/checkpoints',
+                    default='outputs/SNIPS-ALL/20191014-104316/checkpoints',
```

Comment:
doesn't look like a particularly meaningful default - you should eventually remove this